### PR TITLE
Implement trailing stop order for Binance

### PR DESCRIPTION
**Pull request by evzpav**
@beaquant @frankrap I added the trailing stop order type for Binance Futures.
Kindly take a look.

Evandro

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,11 @@ func (s *BasicStrategy) OnTick() error {
 	s.Exchange.OpenLong(symbol, OrderTypeLimit, 5000, 10)
 	s.Exchange.CloseLong(symbol, OrderTypeLimit, 6000, 10)
 
+	//Trailing Stop Market Order - sell stop loss order for long position
+	callbackRate := 5.0 // from 0.1% until 5% allowed
+	s.Exchange.PlaceOrder(symbol, Sell, OrderTypeTrailingStopMarket, 0.0, 10,
+		OrderCallbackRateOption(callbackRate), OrderReduceOnlyOption(true))
```
Comment (ksidenko) [inline README.md:91]: Also **OrderActivationPriceOption** with activation price pass needed. see https://binance-docs.github.io/apidocs/futures/en/#new-order-trade → activationPrice

**Comment by ksidenko** [comment]
@evzpav just update readme.md and I merge PR

**Comment by ksidenko** [comment]
also pls update the branch to the latest master

**Comment by evzpav** [comment]
Thank you @ksidenko !

File: README.md (hunk 1 above)
Comment (evzpav) [reply README.md]: @ksidenko actually `activationPrice` is optional. Default is latest price: "Used with TRAILING_STOP_MARKET orders, default as the latest price(supporting different workingType)". I tested without it and ti works fine. 
But I'll include it here in the README.

**Comment by evzpav** [comment]
> @evzpav just update readme.md and I merge PR

Thanks for reviewing!

**Pull request merged**